### ISSUE TITLE
medpy is NOT compatible with LATEST pydicom?

### ISSUE THREAD

**Issue by jiapei100** [loli/medpy#85]
After a trivial modification from **dicom** to **pydicom** in 2 files,

```
io/load.py
io/header.py
```

I still got the following 2 **ERROR** messages:

```console
➜  Medical python 02_medpy.py
Traceback (most recent call last):
  File "~/.local/lib/python3.6/site-packages/medpy/io/header.py", line 95, in get_offset
    return __get_offset_pydicom(hdr)
  File "~/.local/lib/python3.6/site-packages/medpy/io/header.py", line 269, in __get_offset_pydicom
    return tuple(map(float, hdr.ImagePositionPatient)[:2])
TypeError: 'map' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "02_medpy.py", line 14, in <module>
    header.get_offset(image_header)
  File "~/.local/lib/python3.6/site-packages/medpy/io/header.py", line 101, in get_offset
    raise AttributeError('The provided header {} is of unknown type or does not support queries for offsets.'.format(type(hdr)))        
AttributeError: The provided header <class 'pydicom.dataset.FileDataset'> is of unknown type or does not support queries for offsets.
```

Any plan to upgrade **medpy** to be compatible with **pydicom 1.2.2** ??

Cheers

**Comment by loli** [comment]
MedPy 0.3.0 from PyPi is not compatible with Python3.

Please pull the current master branch from GitHub, which is transitioning to Python3 right now. And thanks for reporting the error, this one escaped us before (fixed with a1e7d11).

Let me now if you encounter any further errors.

**Comment by loli** [comment]
Least required version should be set to: https://github.com/pydicom/pydicom/releases/tag/v1.0.2

**Comment by loli** [comment]
Fixed with 7e9c0b4
